tests/idGenerator.test.ts

```
@@ -0,0 +1,144 @@
+import { generateId, generateCorrelationId } from "../../src/utils/idGenerator";
+
+// ---------------------------------------------------------------------------
+// Helpers
+// ---------------------------------------------------------------------------
+
+/** Standard UUID regex: xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx */
+const UUID_RE =
+  /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/;
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)